apps/src/music/views/SoundsPanel.tsx

```diff
@@ -105,34 +128,29 @@ const SoundsPanelRow = ({
         </div>
         <div className={styles.previewContainer}>
           <FontAwesome
+            title={undefined}
```

Comment:
Oh nothing, TS just complains because it thinks it's a required prop 🙂 ideally I'd use the `FontAwesomeV6Icon` component, but it currently doesn't accept an onClick (this should probably actually be a button component wrapping an icon) so I figured we could address it as a follow up.